website/content/docs/concepts/client-count/client-usage.mdx

```diff
@@ -29,7 +29,7 @@ need to enable usage metrics since it is disabled by default.
 
 1. Select **Edit configuration**.
 
-   ![Edit configuration](/img/ui-usage-metrics-config.png)
+   ![Edit configuration](/img/ui-client-usage-config.png)
```

Comment:
Alt text should be useful and informative. "Edit configuration" doesn't tell me anything about what the screenshot shows